tests/integration/test_node_port.py

```diff
@@ -132,15 +132,14 @@ async def test_build_and_deploy(ops_test: OpsTest, pgb_charm):
 async def test_node_port_and_clusterip_setup(ops_test: OpsTest):
     """Test the nodeport."""
     # Test the writes to the database using the client app
-    psql_app = ops_test.model.applications.get(POSTGRESQL_APP_NAME)
-    await start_continuous_writes(ops_test, psql_app.name)
+    await start_continuous_writes(ops_test, PGB)
```

Comment:
Interface has limit so can't also connect to PG. If we want the continuous_writes to go directly to PG instead of through PGB, we should add another test app.

Reply:
deploy second app for node port. test app is light.